test/e2e/framework/managed/federation.go

```diff
@@ -80,6 +80,30 @@ func SetUpFederationFixture(tl common.TestLogger, clusterCount int) *FederationF
 	return f
 }
 
+func EnableTypeConfigResource(name, namespace string, config *rest.Config, tl common.TestLogger) {
+	for _, federateDirective := range loadFederateDirectives(tl) {
+		resources, err := federate.GetResources(config, federateDirective)
+		if err != nil {
+			tl.Fatalf("Error retrieving resource definitions for FederateDirective %q: %v", federateDirective.Name, err)
+		}
+
+		if federateDirective.Name == name {
+			err = federate.CreateResources(nil, config, resources, namespace)
+			if err != nil {
+				tl.Fatalf("Error creating resources for FederateDirective %q: %v", federateDirective.Name, err)
+			}
+		}
+	}
+}
+
+func DeleteTypeConfigResource(name, namespace string, config *rest.Config, tl common.TestLogger) {
```

Comment:
s/Delete/Disable/

Reply:
`Delete` indicates `true` in below call to delete federated resource.
```
    err := federate.DisableFederation(nil, config, qualifiedName, true, false)
```